d3-geo/index.d.ts

```diff
@@ -936,6 +936,16 @@ export interface GeoPath<This, DatumObject extends GeoPermissibleObjects> {
     centroid(object: DatumObject): [number, number];
 
     /**
+     * Returns the projected planar length (typically in pixels) for the specified GeoJSON object.
+     * Point and MultiPoint features have zero length. For Polygon and MultiPolygon features, this method computes the summed length of all rings.
+     *
+     * This method observes any clipping performed by the projection.
+     *
+     * @param object GeoJSON object to measure.
```

Comment:
I actually changed the whole parameter comment for it to match the sentence structure of the other ones.